servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -184,64 +182,54 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
   /***
    * Gets the {@link UIFileInfo} object based on file id.
    *
-   * @param tachyonFileSystem the {@link TachyonFileSystem} client
+   * @param fileSystem the {@link tachyon.client.file.FileSystem} client
    * @param fileId the file id of the file
    * @return the {@link UIFileInfo} object of the file
    * @throws FileDoesNotExistException if the file does not exist
    * @throws IOException if an I/O error occurs
    */
-  private UIFileInfo getUiFileInfo(TachyonFileSystem tachyonFileSystem, long fileId)
+  private UIFileInfo getUiFileInfo(FileSystem fileSystem, long fileId)
       throws FileDoesNotExistException, BlockDoesNotExistException, IOException, TachyonException {
-    return getUiFileInfo(tachyonFileSystem, fileId, TachyonURI.EMPTY_URI);
+    return getUiFileInfo(fileSystem, fileId, TachyonURI.EMPTY_URI);
   }
 
   /***
    * Gets the {@link UIFileInfo} object based on file path.
    *
-   * @param tachyonFileSystem the {@link TachyonFileSystem} client
+   * @param fileSystem the {@link tachyon.client.file.FileSystem} client
    * @param filePath the path of the file
    * @return the {@link UIFileInfo} object of the file
    * @throws FileDoesNotExistException if the file does not exist
    * @throws IOException if an I/O error occurs
    */
-  private UIFileInfo getUiFileInfo(TachyonFileSystem tachyonFileSystem, TachyonURI filePath)
+  private UIFileInfo getUiFileInfo(FileSystem fileSystem, TachyonURI filePath)
       throws FileDoesNotExistException, BlockDoesNotExistException, IOException, TachyonException {
-    return getUiFileInfo(tachyonFileSystem, -1, filePath);
+    return getUiFileInfo(fileSystem, -1, filePath);
   }
 
   /**
    * Gets the {@link UIFileInfo} object that represents the file id, or the file path if file id is
    * -1.
    *
-   * @param tachyonFileSystem the {@link TachyonFileSystem} client
+   * @param fileSystem the {@link tachyon.client.file.FileSystem} client
    * @param fileId the file id of the file
    * @param filePath the path of the file. valid iff fileId is -1
    * @return the {@link UIFileInfo} object of the file
    * @throws FileDoesNotExistException if the file does not exist
    * @throws IOException if an I/O error occurs
    */
-  private UIFileInfo getUiFileInfo(TachyonFileSystem tachyonFileSystem, long fileId,
+  private UIFileInfo getUiFileInfo(FileSystem fileSystem, long fileId,
```

Comment:
Updated, currently its a very basic solution, I've opened a ticket for a more long term solution. https://tachyon.atlassian.net/browse/TACHYON-1525